lib/notification.dart

```diff
@@ -93,48 +142,64 @@ class Notification {
 
     try {
       await localNotif.zonedSchedule(
-        alarmId,
-        title,
-        body,
-        zdt,
-        platformChannelSpecifics,
-        androidAllowWhileIdle: true,
-        uiLocalNotificationDateInterpretation:
-            UILocalNotificationDateInterpretation.absoluteTime,
-      );
+          alarmId, title, body, zdt, platformChannelSpecifics,
+          androidAllowWhileIdle: true,
+          uiLocalNotificationDateInterpretation:
+          UILocalNotificationDateInterpretation.absoluteTime,
+          payload: '$title ,, $body');
```

Comment:
Please put a comma after the last parameter in order to have a line jump. It improves code readability.